torch/testing/_internal/common_optimizers.py

```diff
@@ -1027,6 +1047,13 @@ def _get_optim_inputs_including_global_cliquey_kwargs(
                 "TestOptimRenewed",
                 "test_forloop_goes_right_direction_multigpu",
             ),
+            DecorateInfo(
+                skipIfTorchDynamo(
+                    "No closure handling, https://github.com/pytorch/pytorch/issues/116494"
```

Comment:
This issue is closed. So this should work now? :D

Reply:
It does, actually! Updated with newer bug + removed for everyone else.